packages/hackmcx-client/src/LandingPage.js

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import { CssBaseline } from '@mui/material';
+import { IconButton } from '@mui/material';
+import Button from '@material-ui/core/Button';
+import { Box } from '@mui/system';
+import Grid from '@material-ui/core/Grid';
+import { BrowserRouter as Router, Route, Routes, Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const LandingPage = () => {
+	const [example, setExample] = useState('-_-');
+	useEffect(async () => {
+		await axios
+			.get('http://localhost:3003/api/vi/posts')
+			.then((response) => {
+				setExample(response.data[0].CURRENT_TIMESTAMP);
```

Comment:
response.data[0].CURRENT_TIMESTAMP will not get anything

Try:
response.data[0].imageUrl